impl/e2e-tests/normalized-aggregation-service-url.json

```diff
@@ -0,0 +1,14 @@
+{
+  "events": [
+    {
+      "seconds": 1,
+      "site": "advertiser.example",
+      "event": "measureConversion",
+      "options": {
+        "aggregationService": "https://AGG-SERVICE.example",
```

Comment:
Hmm, I'm not sure that I like the way this is turning out.  Shouldn't we be doing simple string comparison for these?  Otherwise, we are exposed to URL normalization for interoperability.